plugin/register.vim

```diff
@@ -39,3 +39,4 @@ call s:registry.AddExtension(codefmt#shfmt#GetFormatter())
 call s:registry.AddExtension(codefmt#zprint#GetFormatter())
 call s:registry.AddExtension(codefmt#fish_indent#GetFormatter())
 call s:registry.AddExtension(codefmt#nixpkgs_fmt#GetFormatter())
+call s:registry.AddExtension(codefmt#luaformatterfiveone#GetFormatter())
```

Comment:
Can you add it to the list of formatters at https://github.com/google/vim-codefmt/blob/6761a211eb8690e84167a1cb2efb11f31e07973d/autoload/codefmt.vim#L28 and see if you can get the helpfile regenerated so it shows up as an available formatter in the docs?

Reply:
Done